docusaurus/docs/Android/03-ui/04-components/02-channel-list-header.md

```diff
@@ -1,17 +1,50 @@
 # Channel List Header
 
-<!-- TODO: Import whatever makes sense to import from https://getstream.io/chat/docs/android/channel_list_header_view/?language=kotlin -->
-
 ## Overview
 
-<!-- TODO: Brief description and a couple screenshots with default styling. -->
+`ChannelListHeaderView` is a component designed to be displayed at the channels screen of your app. 
```

Comment:
I see where I should put the View, but not what it does, which I fell that is missing here. Something like: "This is the view that shows the connection status, the title of the channel or conversation name..." would be nice.

Reply:
Explained this more precisely